pkg/apis/litmuschaos/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -89,6 +89,7 @@ func (in *ChaosEngineList) DeepCopyObject() runtime.Object {
 func (in *ChaosEngineSpec) DeepCopyInto(out *ChaosEngineSpec) {
 	*out = *in
 	out.Appinfo = in.Appinfo
+	out.Components = in.Components
```

Comment:
These changes are written by you or it's auto-generated?

Reply:
autogenerated